create-rust-app_cli/src/content/project.rs

```diff
@@ -122,6 +122,7 @@ pub struct CreationOptions {
     pub cra_enabled_features: Vec<String>,
     pub backend_framework: BackendFramework,
     pub backend_database: BackendDatabase,
+    pub cli_mode: bool,
```

Comment:
add cli_mode to the CreationOptions struct so that we don't ask for missing input in cli_mode (just fail instead)